Cargo.toml

```diff
@@ -31,6 +31,8 @@ overflow-checks = true
 backtraces = ["cosmwasm-std/backtraces"]
 # use library feature to disable all instantiate/execute/query exports
 library = []
+# enable-test-utils allows the test classes to expose shared code amongst each other
+enable-test-utils = []
```

Comment:
This allows us to have shared test code between files, but also not include that code as a primary functionality for others downloading these libs.  Obviously no one actually DOES download these libs, but it's good practice